api/draft_registrations/views.py

```diff
@@ -50,10 +50,13 @@ class DraftRegistrationList(NodeDraftRegistrationsList):
     )
 
     serializer_class = DraftRegistrationSerializer
-
     view_category = 'draft_registrations'
     view_name = 'draft-registration-list'
 
+    # overrides NodeDraftRegistrationList
+    def get_serializer_class(self):
+        return DraftRegistrationSerializer
+
```

Comment:
Do you need L52 or L77 with these changes?

Reply:
Removed